crates/core/src/node/batch/factory.rs

```diff
@@ -85,7 +85,7 @@ pub struct MainBatchExecutorFactory<Tr> {
     /// Note that this flag, if set to `true`, is strictly more permissive than if set to `false`. It means
     /// that in cases where the node is expected to process any transactions processed by the sequencer
     /// regardless of its configuration, this flag should be set to `true`.
-    optional_bytecode_compression: bool,
+    enforced_bytecode_compression: bool,
```

Comment:
Could have skipped this file as it is copy-pasted from zksync-era but doesn't really matter